source/includes/tpf/data.md

```diff
@@ -91,6 +91,16 @@ Data resulting from this transformation describe dataset structure and contents
 OpenDataSoft datasets RDF structure can be edited using the RDF Mapping language ([RML](http://rml.io/)).
 Editing the RML mapping of a dataset modifies the RDF result returned by TPF API and RDF exports and updates the `Classes` and `Properties` filters.
 
+A dereferenceable URI returns informations about the resource identified by the URI. Dereferenceable URI's are one of the [Principles of Linked Data](https://en.wikipedia.org/wiki/Linked_data). OpenDataSoft proposes a service that makes URI's dereferenceable and always up to date. To use this service, resource URI have to fulfill the following template:
```

Comment:
Maybe for better clarity you could say something like:

"OpenDataSoft provides an endpoint that accepts dereferenced URIs according to the [Principles of Linked Data](https://en.wikipedia.org/wiki/Linked_data). 

An URI is said to be dereferenceable if it can be accessed to get more information about the object it describes. In other words, this endpoint enables to generate URIs that uniquely describe an OpenDataSoft record, and are guaranteed to lead to the corresponding, up-to-date record.

Resource URIs follow the following format:"

I'm not sure about the resource/record relation, and if you can have dereferenceable URIs for other kinds of OpenDataSoft objects (dataset, field?), so I'll let you rephrase for better accuracy. What do you think?